docker_entrypoint.sh

```diff
@@ -18,4 +18,8 @@ else
   printf "\n${help_result}"
 fi
 
+if [[ -n "${ENABLE_HEAPTRACK}" ]]; then
```

Comment:
```suggestion
if [ -n "${ENABLE_HEAPTRACK:-}" ]; then
```

avoid unbound variables

(arguably this shell script should have `set -u` and `set -o pipefail` as well but that's out of scope for this patch)